app/assets/javascripts/controllers/DragDropCtrl.coffee

```diff
@@ -0,0 +1,85 @@
+# TODO: create modal for save option?
+@bruseApp.controller 'DragDropCtrl', ['$scope', '$http', ($scope, $http) ->
+    $scope.fileObject = null
+    $scope.dataObject = {location:''}
+    $scope.imageFile = null
+    $scope.isDropped = false
+    $scope.isSaved = false
```

Comment:
They get the value from the isolated scope. But perhaps they don't need to be initialized?